topics-and-advance-readings/trust-ux-issues.adoc

```diff
@@ -1,16 +1,41 @@
-= The Threat Model
+= UX Issues in Communicating Threats and Countermeasures
```

Comment:
@genevievePrima We can comment an discuss in-line. For example, here is a proposed new title. What do you think?